app/src/main/java/com/example/abha_create_verify_android/ScannedAadhaarInfoActivity.kt

```diff
@@ -88,14 +76,23 @@ class ScannedAadhaarInfoActivity : ReactActivity() {
         return super.onOptionsItemSelected(item)
     }
 
+    private fun handleBack() {
+        if(Variables.isCreateAbhaScan) {
+            val intent = Intent(this, DemographicsManualOrQRScanActivity::class.java)
+            startActivity(intent)
+            finish()
+        }
```

Comment:
can we remove this else and just add finish outside?

Reply:
done